requirements.txt

```diff
@@ -23,7 +23,8 @@ tzlocal
 wait_for
 websocket_client
 dateparser
-boto3
+boto3>=1.6.6
```

Comment:
I'd much prefer a frozen version here, because we've been seeing failures from dependent package updates much too often.